### ISSUE TITLE
Search for package descriptions that still have "stub description" in them and write proper descriptions

### ISSUE THREAD

**Issue by KyleAMathews** [gatsbyjs/gatsby#5073]

**Comment by ryanditjia** [comment]
I could only find https://github.com/gatsbyjs/gatsby/pull/5083 that has stub and is actually working. None of the stub packages have working code (aphrodite, cxs, docx, etc). Or am I looking at the wrong branch (am on master)?

**Comment by m-allanson** [comment]
Thanks for PR #5083 @ryanditjia!

cc @KyleAMathews, were there any other stub descriptions that you noticed?

**Comment by mikelax** [comment]
Here are a few stub pages that I found:

- https://next.gatsbyjs.org/docs/v0-release-notes/
- https://next.gatsbyjs.org/docs/v2-release-notes/
- https://next.gatsbyjs.org/docs/overview/
- https://next.gatsbyjs.org/packages/gatsby-plugin-react-next/?= -  broken page, should this be removed?

**Comment by KyleAMathews** [comment]
Fixed in https://github.com/gatsbyjs/gatsby/pull/8305

**Comment by KyleAMathews** [comment]
actually it's not 🤦‍♂️

**Comment by KyleAMathews** [comment]
This issue is about writing proper descriptions for the 10 packages that show up in this search https://www.gatsbyjs.org/plugins/?=stub%20des

**Comment by siakaramalegos** [comment]
I can take a pass at this.

**Comment by siakaramalegos** [comment]
So I think I have a fix for this, but I'm having a hard time verifying because running `gatsby develop` in `www` is giving some really weird path and other errors.
![image](https://user-images.githubusercontent.com/4777393/46302406-99127c00-c56e-11e8-9a81-0c899cd2aa52.png)

**Comment by siakaramalegos** [comment]
It's like it's trying to use absolute paths instead of relative. Lots more errors before those also. I can submit a PR (it's just json file I updated), but I'd prefer validating it worked first.

**Comment by siakaramalegos** [comment]
I think part of the problem is that I was following the contributing instructions blindly - my local gatsby-cli is behind the one in the repo. I'll create another issue to update the instructions to run `npm run develop` instead of `gatsby develop`. The CLI should probably check for a locally installed one first though right?

Also, my problems just changed - I still can't get develop to work. The very first error I get is `TypeError: Cannot read property 'allMarkdownRemark' of undefined`

**Comment by siakaramalegos** [comment]
It looks like the `www` site is broken right now, but I am able to load the plugins page. I was assuming `react-instantsearch` was looking in the core /packages/ directory but I think instead it is pointing to something hosted with Algolia? https://community.algolia.com/react-instantsearch/ I cannot see the Algolia set up to know if my changes in the above PR will update the indexed plugins list on Algolia's website.

I'll put this comment in the PR as well. I think someone with access or knowledge will have to verify. Maybe Josh Weaver? He wrote a blog post about setting it up.

**Comment by DSchau** [comment]
@siakaramalegos 

> It looks like the www site is broken right now

Whoa, what do you mean by this?

**Comment by siakaramalegos** [comment]
Uh, are more plugins/transformers being created with the stub description? My PR cleared all the ones as of this morning but now there are 4 new ones.

**Comment by siakaramalegos** [comment]
@DSchau It's still broken - the main index page as well as some other pages are broken in development at least.

**Comment by DSchau** [comment]
@siakaramalegos running into something like this?

```bash
success update schema — 0.714 s
GraphQL Error Unknown field `allDependencies` on type `starterShowcase_2`

  file: /Users/dschau/Code/Work/gatsby/gatsby/www/src/pages/starters.js

   8 |             starterShowcase {
   9 |               slug
  10 |               stub
  11 |               description
  12 |               stars
  13 |               lastUpdated
  14 |               owner
  15 |               name
  16 |               githubFullName
  17 |               gatsbyMajorVersion
> 18 |               allDependencies
     |               ^
  19 |               gatsbyDependencies
  20 |               miscDependencies
  21 |             }
  22 |           }
  23 |           url
  24 |           repo
  25 |           description
  26 |           tags
  27 |           features
  28 |           internal {

```

**Comment by siakaramalegos** [comment]
@DSchau Yes, but I don't think that's the only or main problem. About 30 errors when you open localhost:8000

**Comment by siakaramalegos** [comment]
Oh that is interesting though - this morning the error was on `gatsbyMajorVersion`, the previous item in the list.

**Comment by LeKoArts** [comment]
@siakaramalegos The solution is a missing .env file 😆 
https://github.com/gatsbyjs/gatsby/tree/master/www#working-with-the-starter-showcase

**Comment by siakaramalegos** [comment]
@LeKoArts I'm not working on the starter showcase (?)

**Comment by DSchau** [comment]
@siakaramalegos failing GraphQL queries in one part of the site (i.e. starter showcase) have ramifications elsewhere. So queries failing there because of missing fields, because of missing env var, cause the site to fail hard 😅 I've addressed it as best I can (for now!) in the above PR.

**Comment by amberleyromo** [comment]
Mostly closed in #8677.

There are a few remaining items with stub descriptions that are published plugins whose source has been deleted (no longer in the packages dir). Did these get relocated?

<img width="1455" alt="screen shot 2018-10-04 at 5 03 12 pm" src="https://user-images.githubusercontent.com/3461087/46506785-05091480-c7fb-11e8-8178-24a0010f08f7.png">

**Comment by KyleAMathews** [comment]
@amberleyromo just deprecated the remaining two packages that showed up (which we filter out of results) so closing this!
